src/tests/ngraph_helpers/snippets_ngraph_functions/src/subgraph_permute.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
could you please rename the file: `permute` -> `transpose`

Reply:
No problem